librz/core/cmd_debug.c

```diff
@@ -832,29 +832,38 @@ static int step_until_inst(RzCore *core, const char *instr, bool regex) {
 	return true;
 }
 
-static int step_until_optype(RzCore *core, const char *_optypes) {
+static void dbg_follow_seek_register(RzCore *core) {
+	int follow = rz_config_get_i(core->config, "dbg.follow");
+	rz_cons_break_pop();
+	if (follow > 0) {
+		ut64 pc = rz_debug_reg_get(core->dbg, "PC");
+		if ((pc < core->offset) || (pc > (core->offset + follow))) {
+			rz_core_cmd0(core, "sr PC");
```

Comment:
why this instead of `rz_core_seek_to_register` as it originally was?

Reply:
My mistake. I had copied it from the PR (where rz_core_cmd0 was used), not from the latest `dev`.